src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -200,19 +200,25 @@ internal sealed partial class ConsoleHost
                 // The JumpList does persist as long as the filepath of the executable does not change but there
                 // could be disruptions to it like e.g. the bi-annual Windows update, we decided to
                 // not over-optimize this and always create the JumpList as a non-blocking background task instead.
-                Task.Run(() =>
+                try
                 {
-                    try
+                    // APIs are STA only and until issue 7216 is not resolved, PowerShell operates still in MTA mode.
+                    var thread = new Thread(() =>
```

Comment:
This new STA thread creation should be moved into `CreateElevatedEntry`. In `ConsoleHost.cs`, we should just call `TaskbarJumpList.CreateElevatedEntry` and that's it.

Reply:
Because we have PR to return STA startup parameter we should follow https://referencesource.microsoft.com/#System/services/monitoring/system/diagnosticts/Process.cs,2850

Reply:
Ok, moved logic into `TaskBarJumpList.cs` and added logic to detect current appartmentstate.